numpy/core/_add_newdocs.py

```diff
@@ -1036,7 +1036,12 @@
         A string containing the data.
     dtype : data-type, optional
         The data type of the array; default: float.  For binary input data,
-        the data must be in exactly this format.
+        the data must be in exactly this format.  Supported dtypes are
```

Comment:
Maybe we can just say that most builtin numeric types are supported and extension types may be supported?